src/main/kotlin/org/onflow/flow/sdk/models.kt

```diff
@@ -675,6 +675,18 @@ data class FlowBlock(
     }
 }
 
+data class ExecutionResult(
+    val id: FlowId,
+    val parentId: FlowId
```

Comment:
There's a few missing fields here https://github.com/onflow/flow/blob/3d7cfcfd52ac2532ebe244f9f66ecc8d76bd9fed/protobuf/flow/entities/execution_result.proto#L9-L12

We are missing:
 - chunks
 - serviceEvents